<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="grB_programm">
      <property name="title">
       <string>Программа</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,20,10,20,0,0,0" columnstretch="35,65">
       <item row="6" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_ButtonOkClear" stretch="0,0">
         <item>
          <widget class="QPushButton" name="pb_calculate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Рассчитать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_clear">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Очистить все</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" rowspan="6">
        <widget class="QGroupBox" name="grB_file">
         <property name="title">
          <string>Просмотр файла</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="lw_file">
            <property name="font">
             <font>
              <family>Lucida Console</family>
              <pointsize>9</pointsize>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerItem</enum>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
            <property name="uniformItemSizes">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grB_progress">
            <property name="title">
             <string>Прогресс</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="lb_progress">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="grB_settings">
         <property name="title">
          <string>Установки</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lb_stopFrame">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Кадр останова:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="MySpinBox" name="spB_stopFrame">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>2000000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_path">
         <item>
          <widget class="QLabel" name="lb_progPath">
           <property name="text">
            <string>Путь к программе:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_pathFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="pb_loadFile">
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_findFile">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="grB_info">
         <property name="title">
          <string>Информация</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="1">
           <widget class="QLabel" name="lb_countOfFrames">
            <property name="minimumSize">
             <size>
              <width>114</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lb_toolName">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Инструмент:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lb_countOfFramesName">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Количество кадров:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lb_tool">
            <property name="minimumSize">
             <size>
              <width>114</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="grB_typeOfProcessing">
            <property name="title">
             <string>Виды обработок</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QListWidget" name="lw_typeOfProcessing"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="grB_findFrame">
         <property name="title">
          <string>Поиск кадра</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="60,0,0">
          <item>
           <widget class="MySpinBox" name="spB_findFrame">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_fundFrame">
            <property name="text">
             <string>Найти</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mb_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mb_file_exit"/>
   </widget>
   <widget class="QMenu" name="mb_help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="mb_help_about"/>
   </widget>
   <addaction name="mb_file"/>
   <addaction name="mb_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mb_help_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="mb_file_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header>myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
